README.md

```diff
@@ -34,6 +34,31 @@ BTCPay Server's Docker deployment simplifies the setup by automatically configur
 # For maintainers
 
 ## Building and testing
+
+## 🧑‍💻 Local Development Setup
+If you're contributing to this plugin or running a local development instance of BTCPay Server with the Monero plugin, follow these steps.
+## 1) Requirements
+
+- .NET 8.0 SDK or later
+- JetBrains Rider (recommended) or Visual Studio Code with C# support
+- Git
+- Docker and Docker Compose
+
+## 2) Clone the Repositories
+Create a working directory and clone both the BTCPay Server and Monero plugin repositories side by side:
+If you are a developer maintaining this plugin, in order to maintain this plugin, you need to clone this repository with `--recurse-submodules`:
+
+```bash
+git clone https://github.com/btcpayserver/btcpayserver
+git clone --recurse-submodules https://github.com/btcpay-monero/btcpayserver-monero-plugin
```

Comment:
@deverickapollo isnt btcpay a submodule of the plugin (negating the need to clone it separately)?